From: Alex Gonzalez <alexg@balena.io>
Date: Tue, 17 Sep 2024 14:17:05 +0200
Subject: [PATCH] configs/cl-imx8m-mini_defconfig: increase the default
 environment size

Recent BSP changes have moved this setting from a iot-gate-imx8 specific
configuration to the compulab COM defconfig.

Changelog-entry: Increase the default environment size
Signed-off-by: Alex Gonzalez <alexg@balena.io>
---
 configs/cl-imx8m-mini_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/cl-imx8m-mini_defconfig b/configs/cl-imx8m-mini_defconfig
index 6d735ec8862b..10c511016360 100644
--- a/configs/cl-imx8m-mini_defconfig
+++ b/configs/cl-imx8m-mini_defconfig
@@ -4,7 +4,7 @@ CONFIG_SPL_SYS_DCACHE_OFF=y
 CONFIG_ARCH_IMX8M=y
 CONFIG_SYS_TEXT_BASE=0x40200000
 CONFIG_SPL_GPIO_SUPPORT=y
-CONFIG_ENV_SIZE=0x1000
+CONFIG_ENV_SIZE=0x5000
 CONFIG_ENV_OFFSET=0x4400
 CONFIG_DM_GPIO=y
 CONFIG_SPL_MMC_SUPPORT=y
